### DISCUSSION TITLE
Characters Idling On Sex

### DISCUSSION THREAD

**Discussion by ShylokVakarian** [ErinaSugino/Starbound-SBR-Mod#90] [Trouble Shooting] [answered]
Instead of sexing, they're idling.  No animations, no positions appear in the wheel for the brief moment it appears, nothing.

[starbound.log](https://github.com/ErinaSugino/Starbound-SBR-Mod/files/14315708/starbound.log)

**Comment by ErinaSugino** [comment]
https://github.com/ErinaSugino/Starbound-SBR-Mod/wiki/Advanced-Troubleshooting

**Comment by ShylokVakarian** [comment]
I did read that over.  Did I fuckin' miss something, or...?

**Comment by ErinaSugino** [comment]
You missed the part where it says the right "Teratophilia Scenes" version is the file starting with "SxB_". You have the old version of that mod which completely explodes the UI.

**Comment by ShylokVakarian** [comment]
I just friggin' downloaded that, from the exact mod page the thing linked to.  How do I still not have the right version?

Hang on, lemme try this again.

**Comment by ShylokVakarian** [comment]
Nope, still have the problem.  UI isn't exploded anymore, but still no smexing.  Anything else I clearly missed?

[EDIT: Hang on, found where to enable the logging, will get a new log]

EDIT: And, yes, you may call me an idiot.

**Comment by ErinaSugino** [comment, marked as answer]
No related errors anymore.
Keep in mind that as the player you have to choose a position first for any sex to happen.
Also keep in mind, that positions only become available when they make sense. Worst case scenario on FxF you only have "facesitting" available currently, and that position does not exist for e.g. monsters.

**Comment by ShylokVakarian** [comment]
Alright, that's new, needing to pick a position first.  FxF isn't a problem because I'm gay and not doing that shit anyways.

After going in to try to get a new log with the debug and info options turned on, I've got it working.

**Reply by ErinaSugino** [reply]
It is new, indeed. Idling has been turned into its own state as to allow the potential for solo-actor positions to exist. It also gives the player more control over what they want to do, instead of immediately throwing them into a random sex position. Especially for people into RP.

**Reply by ShylokVakarian** [reply]
A setting to throw them into a random position anyway would be appreciated, especially when there's only one animation anyways (sex toys, primarily, but also occurs with some monsters)

**Comment by ShylokVakarian** [comment]
I also apologize for being an idiot and being too liberal with my skimming.
